lib/galaxy/util/__init__.py

```diff
@@ -1551,6 +1554,9 @@ def send_mail(frm, to, subject, body, config, html=None):
         msg.attach(mp_text)
         msg.attach(mp_html)
 
+    if reply_to:
```

Comment:
Does it ever make sense not to do this ? I'd be in favor of not making this configurable

Reply:
I would say not - people can just choose to ignore the header in downstream applications. And if not, I can't see them ever wanting to reply to their Galaxy server.

Reply:
@mvdbeek I have removed configuration